btsql

```diff
@@ -0,0 +1,51 @@
+CREATE DATABASE sqlexcercise;
+
+CREATE TABLE follow (
```

Comment:
nên dùng `create syntax` như sau

CREATE TABLE IF NOT EXISTS follow(
	id int unsigned not null auto_increment,
	primary key(id)
);
phải có khoá chính cho các bảng. `primary key()`
id phải k được âm: `unsigned`
id phải tự động tăng: auto_increment.
áp dụng cho tất cả các bảng còn lại nhé Phú